README.md

```diff
@@ -114,147 +62,66 @@ pub fn render() {
     App::builder(update, view).build_and_start();
 }
 ```
-
-## Building and running
-
-To build your app, run `cargo make build`, and to host on a dev server, run `cargo make serve`.
-
-For a more robust starting setup, check out Martin Kavik's [seed-quickstart-webpack repo](https://github.com/seed-rs/seed-quickstart-webpack).
-
-## Running included examples
-
-To run an example located in the [examples folder](https://github.com/seed-rs/seed/tree/master/examples),
-run `cargo make start example_name`, where you replace `example_name` with the example name. Eg:
-`cargo make start counter`.
-
-Some examples also require to run API server in another terminal window - `cargo make start_server example_name`.
-
-When server(s) are running, open [127.0.0.1:8000](http://127.0.0.1:8000) in your browser.
+- Run other [examples](examples/) with `cargo make start example_name`.
+- Check out [todomvc](examples/todomvc) if you're feeling confident!
```

Comment:
Don't we want to split `Quickstart` section into two sections - `Quickstarts` and `Examples`?